crate/operator/data/cratedb_cluster_last_user_activity-collector.yaml

```diff
@@ -0,0 +1,14 @@
+---
+collector_name: cratedb_cluster_last_user_activity_collector
+# This metric is intended to know if the cluster is used
+# and to calculate for how long it has not been used.
+metrics:
+- help: "Indicates when the last job occured on the cluster."
+  metric_name: cratedb_cluster_last_user_activity
+  query: |
+    SELECT max(ended) AS cratedb_cluster_last_user_activity
+    FROM sys.jobs_log
+    WHERE username NOT IN ('crate', 'system');
+  type: gauge
```

Comment:
Not sure, but maybe use a `counter` https://prometheus.io/docs/concepts/metric_types/ as this is expected to increase over time, unless we travel in time.

Reply:
It's a bit of a black magic, but the [recommendation](https://www.robustperception.io/are-increasing-timestamps-counters-or-gauges/) is to use gauges for time